app/lib/reporting/declarable_duties.rb

```diff
@@ -0,0 +1,241 @@
+module Reporting
+  class DeclarableDuties
+    MEASURE_EAGER = [
+      :base_regulation,
+      :modification_regulation,
+      { quota_definition: :quota_balance_events },
+      { measure_type: :measure_type_description },
+      {
+        measure_components: [
+          { duty_expression: :duty_expression_description },
+          { measurement_unit: %i[measurement_unit_description measurement_unit_abbreviations] },
+          :monetary_unit,
+          :measurement_unit_qualifier,
+        ],
+      },
+      { additional_code: :additional_code_descriptions },
+      { geographical_area: :geographical_area_descriptions },
+      { measure_excluded_geographical_areas: [{ geographical_area: :geographical_area_descriptions }] },
+      :footnotes,
+      :measure_conditions,
+    ].freeze
+
+    GOODS_NOMENCLATURE_EAGER = [
+      {
+        ns_ancestors: [{ ns_measures: MEASURE_EAGER }],
+        ns_descendants: [{ ns_measures: MEASURE_EAGER }],
+        ns_measures: MEASURE_EAGER,
+      },
+      :goods_nomenclature_descriptions,
```

Comment:
Guessing this one isn't needed if the chapter records themselves aren't getting serialized (since they're not declarable)